tests/unit/loading.tcl

```diff
@@ -0,0 +1,42 @@
+start_server [list overrides [list "key-load-delay" 50 loading-process-events-interval-bytes 1024]] {
```

Comment:
The test is failing, i guess we may need a external skip flag.
```suggestion
start_server [list overrides [list "key-load-delay" 50 loading-process-events-interval-bytes 1024] tags [list "external:skip"]] {
```

Reply:
i commited it so that we can see if the CI will be happy or not.

Reply:
It looks like CI is still unhappy but on unrelated tests?